calendar_event_type_color/views/calendar_event_type_views.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record id="view_calendar_event_type_tree" model="ir.ui.view">
+        <field name="model">calendar.event.type</field>
+        <field name="inherit_id" ref="calendar.view_calendar_event_type_tree" />
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="color" widget="color_picker" optional="show" />
+            </field>
+        </field>
+    </record>
```

Comment:
```suggestion
    <record id="view_calendar_event_type_tree" model="ir.ui.view">
        <field name="model">calendar.event.type</field>
        <field name="inherit_id" ref="calendar.view_calendar_event_type_tree" />
        <field name="priority">90</field>
        <field name="arch" type="xml">
            <tree position="inside">
                <field name="color" widget="color_picker" optional="show" />
            </tree>
        </field>
    </record>
```

to play nice with other modules adding new fields to the same view.
this way it's easier to control the position